docs/Custom Commands.md

```diff
@@ -95,7 +96,13 @@ The `ecu` argument is a constant used to filter incoming messages. Some commands
 
 ### OBDCommand.fast
 
-The `fast` argument tells python-OBD whether it is safe to append a `"01"` to the end of the command. This will instruct the adapter to return the first response it recieves, rather than waiting for more (and eventually reaching a timeout). This can speed up requests significantly, and is enabled for most of python-OBDs internal commands. However, for unusual commands, it is safest to leave this disabled.
+The optional `fast` argument tells python-OBD whether it is safe to append a `"01"` to the end of the command. This will instruct the adapter to return the first response it recieves, rather than waiting for more (and eventually reaching a timeout). This can speed up requests significantly, and is enabled for most of python-OBDs internal commands. However, for unusual commands, it is safest to leave this disabled.
```

Comment:
What are you changing on this line?

Reply:
I just added 'optional'. The *optional* `fast` argument...

Reply:
Ah, that's fine. The patch looks good then, just needs to be tested